### PR TITLE
Cu 357jwgk remove function returning widget

### PR DESCRIPTION
**Pull request by yoppysource**
1. remove all functions and change to the widget.
2. reorder some of constructors.

### CODE REVIEW

**Comment by sebatyler** [comment]
Task linked: [CU-357jwgk Remove function returning widget](https://app.clickup.com/t/357jwgk)

**Comment by yoppysource** [comment]
This PR conflict to previous pull request. Thus, I will close it and wait.

**Comment by sebatyler** [comment]
You don't need to close a PR like this from next time. I recommend resolving conflict. Let's discuss this on Friday.

**Comment by sebatyler** [comment]
This PR is almost same with #44.
Maybe changing base branch for #44 to this branch will be better.

File: lib/widgets/body/faq.dart [hunk 1]
```diff
@@ -22,120 +17,148 @@ Star: As an athlete's support token, it is non-tradable for now. However, the wa
...
-      question,
-      style: getDescriptionStyle(
-        fontWeight: FontWeight.w700,
-        color: const Color(0xffFAFAFA),
-      ),
-    );
-  }
+  @override
+  Widget build(BuildContext context) {
+    List<FAQItem> fAQItems = [];
```
Comment (hayeonhee) [inline lib/widgets/body/faq.dart:44]: How about using `faqItems`? I referred to [Style guide for Flutter repo](https://dart.dev/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words).

**Review by hayeonhee — CHANGES_REQUESTED** [review]
Nice work! I think It would be nice to change the name of variable.

File: lib/widgets/body/faq.dart (hunk 1 above)
Comment (sebatyler) [reply lib/widgets/body/faq.dart:44]: I agree with Yeha. faqItems looks better.

Comment (yoppysource) [reply lib/widgets/body/faq.dart:44]: True! Thanks for the feedback!

**Pull request closed without merging**